guide/admin/uploadlogo.md

```diff
@@ -0,0 +1,20 @@
+## User Handbook
+### Uploading a custom logo & favicon
+
+To upload a custom logo to your forum instance requires you to be in the Administrator Control Panel, you can do this by clicking your username to the top right of your screen, and then clicking **Administration**. The example below will show you how this is done:
+
+![Administration Link Screenshot](687474703a2f2f692e696d6775722e636f6d2f575a315a6a78392e676966.gif)
+
+Once you are in the Administration Panel, you'll see some navigation tabs flowing down the left hand side, click on **Appearance** to reveal the configuration page to the right hand side.
+
+![Appearance Screenshot](687474703a2f2f692e696d6775722e636f6d2f736a6a6b455a362e706e67.png)
+
+Under the **Logo** label, you'll find a button which is labelled **Choose an image...**, click that which will bring up the file upload prompt, find the image you wish to upload them click **Open**
```

Comment:
Dismissing this one. Considering this may change per server, it's too much to try to explain to users. Favicon sizes are well recorded online, and users can experiment with logo sizes or just ask on Flarum's discuss.